tools/schemacode/bidsschematools/render/text.py

```diff
@@ -438,7 +440,7 @@ def append_filename_template_legend(text, pdf_format=False):
 
 """
     else:
-        md = MarkdownIt()
+        md = mdit.MarkdownIt()
```

Comment:
Should this be `markdown_it`?

Reply:
it was defined above as `mdit = lazy.load("markdown_it")` so I guess it is technically correct here/now, BUT may be to avoid such confusions it should be
- `import lazy_loader` (to not need to think either it is `lazy_loader` vs `lazy`)
- `markdown_it = lazy.load("markdown_it")` (to similarly minimize cognitive load here)
?

Reply:
Okay, I was definitely not caffeinated enough when I reviewed this PR. Thank you for explaining!

Reply:
1c: oh well, I think we know that conventions are not *gods given* and could change. I understand somewhat wide spread, frequently used and "concise" import conventions  such as `np`, or `sp` but abbreviating not so long into not so short and not so frequently used (once or twice in a file) -- mneh, no point. anyways -- want to keep `lazy` - fine with me.  `markdown_it` -> `mdit` for few uses imho makes little sense to me personally.

Reply:
I did change it to `markdown_it`...